contents/docs/data/persons.mdx

```diff
@@ -55,3 +55,63 @@ Clicking on a user opens their profile and shows all their properties.
 ## Deleting person data with the API
 
 You can also delete persons data via the API. See the [Data Deletion docs](/docs/privacy/data-deletion) for more information.
+
+## Events with person processing
+
+By default all events create or update an associated person profile (see above). This allows for the following features:
+
+- Person profiles
+- Filtering on person properties
+- Merging identities across distinct_ids (ie merging anonymous users with your logged in user)
+- Cohort creation
+- Feature flag targeting on properties
+- Tracking initial UTM tags and referrers over time
+
+However, there are many usecases that don't require this person processing. Because events that don't do person processing are cheaper for us to ingest, the pricing for these events is lower.
+
+Some examples of usecases that often don't require user profiles
+- Tracking a marketing website
+- B2C apps where users don't sign up or log in
+- High volume API server events
+
+### PostHog-js
```

Comment:
```suggestion
- High volume API server events

> For backward compatibility reasons, and to make sure we don't drop person information accidentally, processing events with persons is the default for configs without any option set. 

### PostHog-js
```

I want to very explicitly state why the more expensive type is the default, when our pricing treats the less expensive option (personless) as the default.